### ISSUE TITLE
Sysvinit breaks when using a Centos7 box

### ISSUE THREAD

**Issue by skmichaelson** [bflad/chef-docker#395]
When installing Docker on Centos7 and attempting to use `Chef::Provider::DockerService::Sysvinit` as the provider, the following error occurs:

```
TypeError
---------
no implicit conversion of false into Array

Resource Declaration:
---------------------
# In /tmp/kitchen/cache/cookbooks/docker/libraries/provider_docker_service_sysvinit.rb

29:           service 'docker' do
30:             provider Chef::Provider::Service::Init::Redhat if node['platform_family'] == 'redhat'
31:             provider Chef::Provider::Service::Init::Insserv if node['platform_family'] == 'debian'
32:             supports restart: true, status: true
33:             action [:enable, :start]
34:           end
35:         end
```

**Comment by skmichaelson** [comment]
It looks like `node['platform_family']` returns 'rhel' instead of 'redhat', so the provider is turning up empty and the service resource doesn't like this very much =(

**Comment by someara** [comment]
Fix merged and released in 1.0.12
